modules/bookObj.js

```diff
@@ -0,0 +1,7 @@
+export default class BookObj {
+    constructor (title, author, id) {
+      this.title = title;
+      this.author = author;
+      this.id = id;
+    }
```

Comment:
## Please  use arrow syntax for functions as you did from the rest of the files: I requirest this because you are using ES6

Reply:
Hello @OLIPLICHE, 
From what I understood, arrow functions can not be used as constructors, isn't that the case?

Reply:
Hi @Abe1able. Arrow functions can never be used as constructor functions, it Completly escaped me. Sorry! you can submit your work. It will be approved. the App is working fine.